### DISCUSSION TITLE
centos 8.3 can't build phalcon-3.4.5

### DISCUSSION THREAD

**Discussion by ikool-cn** [phalcon/cphalcon#15601] [Q&A] [answered]
[root@10-7-171-213 build]# ./install 
find . -name \*.gcno -o -name \*.gcda | xargs rm -f
find . -name \*.lo -o -name \*.o | xargs rm -f
find . -name \*.la -o -name \*.a | xargs rm -f
find . -name \*.so | xargs rm -f
find . -name .libs -a -type d|xargs rm -rf
rm -f libphp.la       modules/* libs/*
Cleaning..
Configuring for:
PHP Api Version:         20151012
Zend Module Api No:      20151012
Zend Extension Api No:   320151012
appending configuration tag "CXX" to libtool
mkdir .libs
 gcc -I. -I/root/cphalcon-3.4.5/build/php7/64bits -DPHP_ATOM_INC -I/root/cphalcon-3.4.5/build/php7/64bits/include -I/root/cphalcon-3.4.5/build/php7/64bits/main -I/root/cphalcon-3.4.5/build/php7/64bits -I/usr/local/php/include/php -I/usr/local/php/include/php/main -I/usr/local/php/include/php/TSRM -I/usr/local/php/include/php/Zend -I/usr/local/php/include/php/ext -I/usr/local/php/include/php/ext/date/lib -DPHALCON_RELEASE -DHAVE_CONFIG_H -mtune=native -O2 -fomit-frame-pointer -c /root/cphalcon-3.4.5/build/php7/64bits/phalcon.zep.c  -fPIC -DPIC -o .libs/phalcon.o
gcc: fatal error: Killed signal terminated program cc1
compilation terminated.
make: *** [Makefile:194: phalcon.lo] Error 1
 gcc -I. -I/root/cphalcon-3.4.5/build/php7/64bits -DPHP_ATOM_INC -I/root/cphalcon-3.4.5/build/php7/64bits/include -I/root/cphalcon-3.4.5/build/php7/64bits/main -I/root/cphalcon-3.4.5/build/php7/64bits -I/usr/local/php/include/php -I/usr/local/php/include/php/main -I/usr/local/php/include/php/TSRM -I/usr/local/php/include/php/Zend -I/usr/local/php/include/php/ext -I/usr/local/php/include/php/ext/date/lib -DPHALCON_RELEASE -DHAVE_CONFIG_H -mtune=native -O2 -fomit-frame-pointer -c /root/cphalcon-3.4.5/build/php7/64bits/phalcon.zep.c  -fPIC -DPIC -o .libs/phalcon.o
gcc: fatal error: Killed signal terminated program cc1
compilation terminated.
make: *** [Makefile:194: phalcon.lo] Error 1

Thanks for compiling Phalcon!
Build succeed: Please restart your web server to complete the installation

**Comment by Jeckerson** [comment]
What version of PHP are you using?

**Reply by ikool-cn** [reply]
PHP 7.0.33

**Comment by Jeckerson** [comment, marked as answer]
You are probably running out of memory. Try to create SWAP file with 1-2Gb in case you can't increase memory itself.
